test/fn/upload-custom-translations.spec.js

```diff
@@ -117,11 +118,14 @@ describe('upload-custom-translations', () => {
 
     it('should set default name for unknown language', () => {
       mockTestDir(`unknown-lang`);
+      sinon.replace(log, 'warn', sinon.fake());
       return uploadCustomTranslations()
         .then(() => expectTranslationDocs('qp'))
         .then(() => getTranslationDoc('qp'))
         .then(messagesQp => {
           assert(messagesQp.name === 'TODO: please ask admin to set this in settings UI');
+          assert(log.warn.lastCall.calledWithMatch('\'qp\' is not a recognized ISO 639 language code, please ask admin to set the name'));
+          sinon.restore();
```

Comment:
`sinon.restore` should be called in a `afterEach` so you don't have to include it in every test.

Reply:
Fixed.